web/ui/templates/status.html

```diff
@@ -13,6 +13,26 @@ <h2 id="runtime">Runtime Information</h2>
           <th>Working Directory</th>
           <td>{{.CWD}}</td>
         </tr>
+        <tr>
+          <th>Configuration reload</th>
+          <td>{{if .ReloadConfigSuccess}}Succesfull{{else}}Failed{{end}}</td>
```

Comment:
Can we do some colouring on this to make it stand out?

Reply:
I added changing row color to red
https://www.w3schools.com/bootstrap/bootstrap_tables.asp